packages/v4/src/content/design-guidelines/components/empty-state/empty-state.md

```diff
@@ -3,208 +3,393 @@ id: Empty state
+<br />  
+
+**Secondary button**: Give the user the ability to access additional resources that can help them through the creation process, such as a tour. 
+
+You can also use the extra small empty state variation inside of a card to present the user with another way to add a new card to their view. Place the empty state card where a new card would be added, such as the first (or last) card in the view. Be sure to also include a primary button in the toolbar so that the user can still perform the action even if the empty state is out of view. 
+
+<img src="./img/add-card-empty-state.png" alt="empty state inside of a card" width="1500px"/> 
+
+## Visual design 
+Icons and colors can help enhance users’ visual understanding of empty state messaging.
+
+### Icons
+To maintain consistency, specific icons are mapped to specific empty state scenarios.
+
+Here is when you should use each icon:
+
+|       |Type |Color |Icon |Usage |
+|---------------|------|-----|-----------|------------|
+| <img src="./img/rocket.png" alt="rocket icon" width="35px"/> |First use or get started |Gray (#8A8D90) |fa-rocket | Use in getting started empty states. You can replace this icon if you already have an application-specific graphic that you want to use instead.
+| <img src="./img/search.png" alt="search icon" width="35"/> |Search |Gray (#8A8D90)|fa-search |Use when there is no data to show.
+| <img src="./img/config.png" alt="configuration icon" width="35"/> |Configuration |Gray (#8A8D90) |fa-wrench |Use when configuration is required or there is a configuration issue.
+| <img src="./img/lock.png" alt="lock icon" width="35"/> |No access |Gray (#8A8D90) |fa-lock | Use when the user is not entitled or the user role does not have access.
+| <img src="./img/error.png" alt="error icon" width="35"/> |Error  |Red (#C9190B) |fa-exclamation-circle |Use when there is an inability to get data, there is backend failure, or an error has occurred.
+| <img src="./img/success.png" alt="success icon" width="35"/> |Success |Green (#486b00) |fa-check-circle |Use to indicate that a task or process has been completed successfully.
+| <img src="./img/add.png" alt="add icon" width="35"/> |Addition or creation |Gray (#8A8D90) |fa-plus-circle |Use when something needs to be added or created.
 
-### No data to show
+### Colors 
+Use color intentionally. If you’re not using a custom icon for a specific use case, an icon should be gray by default (except when it’s used as a status icon).
 
-Help users understand why there is no data to display and explain what they can do to move forward. For example, there may be no data to show because the user has a series of filters applied and the system cannot locate a match. It’s also possible that there is no data because all issues are resolved or all tasks are complete.
+## Content 
+Content varies depending on the type of empty state. For specific information on what content to use  for each empty state type, see the earlier **Usage** section. 
```

Comment:
Can you link to the usage section instead of bolding it?

Reply:
Yes! How do I insert a jump link?

Reply:
`[Usage](#usage)`